src/index.coffee

```diff
@@ -173,7 +178,7 @@ module.exports.configure = (options, app, callback) ->
 
     api.setupModels()
 
-    try {requestsToSave, models} = getRequests options.root
+    try {requestsToSave, models} = getRequests options.modelsPath
```

Comment:
`options` is built based on the given parameter, so it should not break (see line 165 to 172). Since one case (no options given in parameter) was not treated, I added a default case to make sure it doesn't crash.